terraform/projects/infra-assets/backup.tf

```
@@ -11,44 +11,38 @@ resource "aws_s3_bucket_versioning" "assets_backup" {
   versioning_configuration { status = "Enabled" }
 }
 
-resource "aws_iam_policy" "backup" {
-  count  = var.aws_environment == "production" ? 1 : 0
-  name   = "govuk-${var.aws_environment}-assets-backup-policy"
-  policy = data.template_file.backup_policy[0].rendered
+resource "aws_s3_bucket_policy" "assets_backup" {
+  count    = var.aws_environment == "production" ? 1 : 0
+  provider = aws.backup
+  bucket   = aws_s3_bucket.assets_backup[0].id
+  policy   = data.aws_iam_policy_document.bucket_is_replication_target[0].json
 }
 
-resource "aws_iam_role" "backup" {
+data "aws_iam_policy_document" "bucket_is_replication_target" {
   count = var.aws_environment == "production" ? 1 : 0
-  name  = "govuk-${var.aws_environment}-assets-backup"
-
-  assume_role_policy = <<POLICY
-{
-  "Version": "2012-10-17",
-  "Statement": [
-    {
-      "Action": "sts:AssumeRole",
-      "Principal": { "Service": "s3.amazonaws.com" },
-      "Effect": "Allow",
-      "Sid": ""
+  statement {
+    sid = "S3RoleInSourceAccountReplicatesObjectsToDestinationBucket"
+    principals {
+      type        = "AWS"
+      identifiers = [local.replication_service_role_in_prod_account]
```

Nit: Couple places you've got this - wondering if you can just reference the role arn directly instead of using a local?
```suggestion
      identifiers = [aws_iam_role.replication.arn]
```